src/organizations/components/RenameOrgOverlay.tsx

```diff
@@ -15,17 +15,31 @@ import {ErrorHandling} from 'src/shared/decorators/errors'
 class RenameOrgOverlay extends PureComponent<
   RouteComponentProps<{orgID: string}>
 > {
+  state = {
+    visible: false,
+  }
+  componentDidMount() {
+    this.setState({visible: true})
```

Comment:
why is this state stuff here? doesn't the handleClose unmount the component? if so, please remove, if not, please ignore

Reply:
its to manage the visible state and expose the onEscape